apps/ledger-live-desktop/tests/page/drawer/send.drawer.ts

```diff
@@ -18,4 +20,14 @@ export class SendDrawer extends Drawer {
     await expect(this.addressValue(tx.accountToCredit.address)).toBeVisible();
     await expect(this.amountValue).toBeVisible();
   }
+
+  @step("Verify Send NFT information")
+  async expectNftInfos(tx: NFTTransaction) {
+    const transactionType = await this.transactionType.textContent();
+    expect(transactionType).toMatch("Sending");
```

Comment:
Improvment for later, maybe we could create a tansaction status type

Reply:
I create a ticket :) https://ledgerhq.atlassian.net/browse/QAA-340